scripts/mitmproxy_metro_bundler.py

```diff
@@ -0,0 +1,8 @@
+# This script is used during development to allow mitmproxy to intercept request on specific ports and make a proxy-pass on localhost
```

Comment:
Needs shebang `#!/usr/bin/env python`

also needs `chmod a+x`

Reply:
It is not an executable. It is a mitmproxy script. An example from the nonprofit repository https://github.com/mitmproxy/mitmproxy/blob/master/examples/simple/add_header.py